Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public static class Lesson4
+    {
+        public static void Demo()
+        {
+            WelcomePrompt();
+            do
+            {
+                PromptUserForStats();
+
+            } while (PromptCalculateNewBMI());
+        }
+        public static bool PromptCalculateNewBMI()
+        {
+            Console.WriteLine("Do you wish to calculate another person? Y/N ");
+            string response = Console.ReadLine();
+            return response.ToLower().StartsWith("y");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884599)